python/across_api/base/ephem.py

```diff
@@ -0,0 +1,186 @@
+# Copyright © 2023 United States Government as represented by the
+# Administrator of the National Aeronautics and Space Administration.
+# All Rights Reserved.
+
+from typing import Optional
+
+import astropy.units as u  # type: ignore
+import numpy as np
+from astropy.constants import R_earth  # type: ignore
+from astropy.coordinates import (  # type: ignore
+    GCRS,
+    TEME,
+    CartesianDifferential,
+    CartesianRepresentation,
+    SkyCoord,
+    get_body,
+)
+from astropy.time import Time  # type: ignore
+from fastapi import HTTPException
+from sgp4.api import Satrec  # type: ignore
+
+from ..base.schema import EphemGetSchema, EphemSchema, TLEEntry
+from .common import ACROSSAPIBase, round_time
+
+
+class EphemBase(ACROSSAPIBase):
```

Comment:
I thought that we were removing the ephemeris endpoint. Why does this still inherit from `ACROSSAPIBase`?

Reply:
The code breaks if I remove it, and it does no harm, so let's leave that in.